src/utilities/util_printlog.c

```diff
@@ -12,33 +12,58 @@ static int
 usage(void)
 {
     static const char *options[] = {"-f", "output to the specified file", "-x",
-      "display key and value items in hexadecimal format", NULL, NULL};
+      "display key and value items in hexadecimal format", "-l", "start and stop the printlog",
```

Comment:
Perhaps instead of "start and stop the printlog" we could say something like "the start LSN from which the log will be printed, optionally the end LSN can also be specified"

Reply:
Sounds good